### DISCUSSION TITLE
Documentation clarification

### DISCUSSION THREAD

**Discussion by mwawrusch** [wevm/wagmi#1590] [Ideas]
Hi guys, 
while reading through the docs I stumbled upon:

```
Force a specific chain id for the request. The wagmi Client's ethers provider must be set up as a [chain-aware function](https://wagmi.sh/react/client#provider-optional) for this to work correctly.
```
the target link does not provide information what is required to set up a provider as a chain aware function, or mention chain awareness at all.

**Comment by tmm** [comment]
[Added a note.](https://wagmi.sh/react/client#provider) Thanks for the feedback!
